src/MoBi.Core/Services/QualificationStepRunnerFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Services;
+using IContainer = OSPSuite.Utility.Container.IContainer;
+
+namespace MoBi.Core.Services
+{
+   public class QualificationStepRunnerFactory :  OSPSuite.Core.Services.QualificationStepRunnerFactory
+   {
+      public QualificationStepRunnerFactory(IContainer container) : base(container)
+      {
+      }
+
+      public override IQualificationStepRunner CreateFor(IQualificationStep qualificationStep)
+      {
+         throw new System.NotImplementedException();
```

Comment:
This will come later, I needed this factory for pulling up some other stuff. It seems like the kind of class that will be implemented as we proceed in Requalification